Src/.editorconfig

```diff
@@ -205,48 +227,26 @@ dotnet_naming_style.begins_with_i.required_suffix =
 dotnet_naming_style.begins_with_i.word_separator = 
 dotnet_naming_style.begins_with_i.capitalization = pascal_case
 
-dotnet_diagnostic.CS1570.severity = silent
-dotnet_diagnostic.CS1572.severity = silent
-dotnet_diagnostic.CS1573.severity = silent
-dotnet_diagnostic.CS1574.severity = silent
-dotnet_diagnostic.CS1591.severity = silent
-dotnet_diagnostic.CS1712.severity = silent
-dotnet_diagnostic.CS1734.severity = silent
+# A method should not follow a class
+dotnet_diagnostic.SA1201.severity = None
 
-# SA1649: File name should match first type name
-dotnet_diagnostic.SA1649.severity = none
+# 'public' members should come before 'private' members
+dotnet_diagnostic.SA1202.severity = None
 
-file_header_template = Copyright (c) Microsoft. All rights reserved.\nLicensed under the MIT license. See LICENSE file in the project root for full license information.
```

Comment:
>Copyright (c) Microsoft. All rights reserved.\nLicensed under the MIT license. See LICENSE file in the project root for full license information. [](http://example.com/codeflow?start=23&length=145)

@schlaman-ms Why are we removing this file header?

Reply:
I put the file_header_template back. Look later in the file.

The reason for the number of changes is that I used the Generate .editorconfig file from the Visual Studio UI in order to have a modern version with the settings that make sense to have.
I have gone back and put lots of stuff back from the original .editorconfig. It's extremely difficult to do a code review on the .editorconfig changes.